app/views/public/api_documentation.html.erb

```diff
@@ -394,6 +395,63 @@
 
 <hr />
 
+<h2 class="h4" id="stake-explorer">Stake Explorer</h2>
+<p>
+  The Stake Accounts endpoint will return all the stake accounts according to provided criteria. Only mainnet network
+  is available for this query.
+</p>
+
+<h3 class="h6">Request:</h3>
+<pre>curl -H "Token: secret-api-token" '<%= @api_path %>/explorer-stake-accounts/mainnet.json'</pre>
+
+<h3 class="h6">Parameter Options:</h3>
+
+<p><strong class="text-muted">`staker=string`</strong> will output all the stake accounts by staker </p>
+<pre>curl -H "Token: secret-api-token" '<%= @api_path %>/explorer-stake-accounts/mainnet.json?staker=mpa4abUkjQoAvPzREkh5Mo75hZhPFQ2FSH6w7dWKuQ5'</pre>
+<p><strong class="text-muted">`withdrawer=string`</strong> will output all the stake accounts by withdrawer </p>
+<pre>curl -H "Token: secret-api-token" '<%= @api_path %>/explorer-stake-accounts/mainnet.json?withdrawer=4ZJhPQAgUseCsWhKvJLTmmRRUV74fdoTpQLNfKoekbPY'</pre>
+<p><strong class="text-muted">`stake_pubkey=string`</strong> will output stake account with the given pubkey </p>
+<pre>curl -H "Token: secret-api-token" '<%= @api_path %>/explorer-stake-accounts/mainnet.json?stake_pubkey=G4R85Eyqrrf5giV3Uino7KjQqMGujJX5wgByvRwQA9En'</pre>
+<p><strong class="text-muted">`vote_account=string`</strong> will output stake accounts delegated to a given vote account </p>
+<pre>curl -H "Token: secret-api-token" '<%= @api_path %>/explorer-stake-accounts/mainnet.json?vote_account=6559KMdiUseNSAkRcK9WcFcNTppoj6jWtKVedpMkBYCn'</pre>
+
+<p><strong class="text-muted">`per=NN`</strong> will limit the results to NN entries per page, defaults to 100.</p>
```

Comment:
I think default should be smaller, 50 or 25. 
Also, it doesn't seem to be working - I always get 25 records, whenever I provide `per` param or not.

Reply:
Also, you should set a maximum possible per param, as we do for other public endpoints, eg. https://github.com/brianlong/www.validators.app/blob/master/app/controllers/api/v1/ping_things_controller.rb#L12 or https://github.com/brianlong/www.validators.app/blob/master/app/controllers/api/v1/account_authority_controller.rb#L11